spec/text-buffer-spec.coffee

```diff
@@ -990,14 +991,16 @@ describe "TextBuffer", ->
 
       it "notifies ::onDidConflict observers", ->
         buffer.setText("a change")
-        buffer.save()
-        buffer.insert([0, 0], "a second change")
-
         handler = jasmine.createSpy('fileChange')
-        writeFileSync(filePath, "a disk change")
-        buffer.onDidConflict handler
+        buffer.save().then ->
+          buffer.insert([0, 0], "a second change")
+          buffer.onDidConflict handler
+          expect(handler.callCount).toBe 0
+          # wait a little bit to not confuse the save change event, with the write file change event
+          setTimeout ( ->
+            writeFileSync(filePath, "a disk change")
+          ), 500
```

Comment:
Actually, with more investigation, I found out the the synchronous save in node emits one change event, but the async one emits two change events!!!! - pending more investigation